src/rust/src/x509/verify.rs

```diff
@@ -43,6 +47,125 @@ impl CryptoOps for PyCryptoOps {
     }
 }
 
+struct FixedPolicy<'a>(Policy<'a, PyCryptoOps>);
```

Comment:
Is there a better name than `FixedPolicy`?

Reply:
Hmm, maybe `BoundPolicy` (to emphasize that we're binding a `Policy` to its `PyCryptoOps`? Or maybe I could just import `Policy` as `_Policy` or similar and rename this to `Policy`.

Reply:
How about `PyCryptoPolicy` :-)

Reply:
https://github.com/pyca/cryptography/pull/9405/commits/35f5e6f8a91c012fc59d3e5d7b3f965f9b8c393b